src/quo/components/banners/alert_banner/schema.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo.components.banners.alert-banner.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:action? {:optional true} [:maybe boolean?]]
+      [:text {:optional true} [:maybe string?]]
```

Comment:
Wondering if there is any point in having the `alert-banner` without `text` property`? Wdyt, maybe we can make it obligatory?